charts/alfresco-repository/templates/configmap-repository.yaml

```diff
@@ -27,6 +27,12 @@ data:
     -Dmessaging.broker.password="$BROKER_PASSWORD"
     -Dindex.subsystem.name="$SEARCH_FLAVOR"
     {{- include "alfresco-repository.search.config" $ | indent 2 }}
+    {{- range .repository.existingSecrets }}
+      {{- if and .key (hasPrefix "property:" .purpose) }}
+        {{- $property := trimPrefix "property:" .purpose }}
+        {{- printf "-D%s=\"$%s\"" $property .key | nindent 4 }}
+      {{- end }}
+    {{- end }}
```

Comment:
not sure having one secret for each property scale well but if that's for a limited number of use case ok

Reply:
I see it as a general way to pass sensitive properties while getting the value from env vars and avoid storing it in a file.
I don't think there would be that many. Is there a particular concern of why that would not scale?